### ISSUE TITLE
Language definition/selector

### ISSUE THREAD

**Issue by Tamas-Toth-ebola** [amaximus/met_alerts_hu#15]
Hi!

First of all I would like to thanks for your great work on this integration! Could be very useful to rely on the Hungarian local informations.

I have only one request/question.

As met.hu currently already has a languaga selector when the results in the corresponding Hun or Eng language also, would it be possible to define it in the config of the integration to avoid discrete attribute string convertion later?

Thanks for all your efforts!

**Comment by amaximus** [comment]
There might be a language selector but the returned data is mixed:
EN:
```
$cat hover.php\?id\=wbhx\&kod\=13\&lng\=en

                <table cellspacing=1 cellpadding=0 class='stbl' width='100%'>
                <tr><th colspan=3>County Pest</th></tr>
                <tr>
                        <td class='row1' width=31><img src='/images/warningb/ts1.gif' border=0></td>
                        <td class='row1' width=27><img src='/images/warningb/w1.gif' border=0></td>
                        <td class='row1' width=170>Zivatar</td>
                </tr>
        </table>
        <div class='kt-friss'><div>Kiadva: 2022-05-03 09:36 (07:36 UTC)</div><div>[wbhx]</div></div>
$
```

HU:
```
$cat hover.php\?id\=wbhx\&kod\=13

                <table cellspacing=1 cellpadding=0 class='stbl' width='100%'>
                <tr><th colspan=3>Pest megye</th></tr>
                <tr>
                        <td class='row1' width=31><img src='/images/warningb/ts1.gif' border=0></td>
                        <td class='row1' width=27><img src='/images/warningb/w1.gif' border=0></td>
                        <td class='row1' width=170>Zivatar</td>
                </tr>
        </table>
        <div class='kt-friss'><div>Kiadva: 2022-05-03 09:36 (07:36 UTC)</div><div>[wbhx]</div></div>(ha_venv3_9) 
$
```

**Comment by Tamas-Toth-ebola** [comment]
Seems, you have to use `wbex` as the ID parameter to get `en` strings everywhere. Check it:

- https://met.hu/idojaras/veszelyjelzes/hover.php?id=wbhx&kod=13&lng=hu
- https://met.hu/idojaras/veszelyjelzes/hover.php?id=wbex&kod=13&lng=en

Seems the third letter in the ID code is a language code: `wbhx` vs. `wbex`.

**Comment by amaximus** [comment]
Thanks for digging into it, I'll update it today.
A REST API would have been preferred .

**Comment by Tamas-Toth-ebola** [comment]
Thank you! Of course HTML post-processing is the least reliable and convinient solution :disappointed:

**Comment by amaximus** [comment]
Recently released version 2.4.0 adds language support as per met.hu

Please update the integration and get back to me with results.

**Comment by Tamas-Toth-ebola** [comment]
It works fine! :champagne: 

![image](https://user-images.githubusercontent.com/25765137/166461458-cf5a4012-3bec-4fa9-b863-94e77fab868f.png)

Thanks for it again!
